src/atoms/Button/index.js

```diff
@@ -40,15 +40,15 @@ function Button( props ) {
         href={href}
         {...linkAttrs}
       >
-        { icon && <Icon icon={icon} className={styles['icon']} /> }
+        { icon && <Icon icon={icon} renderSVGDOM className={styles['icon']} /> }
```

Comment:
I'm curious to hear people's opinion on this. With the `renderSVGDOM` prop set here, the svg will be rendered directly into the DOM, as opposed to the other approach which used an `<img>` tag with a `src` of the svg. There's no negative network implications of this (since the svg would have to be downloaded either way anyway), but I'm curious if there are any reasons why this approach was initially avoided. There could have maybe been performance implications with SVGs in older versions of React (esp large ones), but I don't think it should matter too much.

Sadly, there's no real scalable way to apply color to SVGs when they're brought in via an `<img>` tag. There's the (well-supported) CSS `filter` property which can be used to apply coloring, but the API isn't exactly what we want. You can't just specify a hex value, you need to figure out how to represent your color in terms of CSS blend properties, and even that only works if you can assume that the SVG will always be loaded in as black (which isn't the case for all of our icons).

Reply:
I thin this _should_ be fine, but it may cause issues with testing because the `renderSVGDOM` prop causes [a network request](https://github.com/policygenius/athenaeum/blob/master/src/atoms/Icon/index.js#L28), which I've had to mock out in the past.

I would say that's probably an OK trade-off since icons in buttons are not used _super_ frequently. Based on your comment, I would prefer to render the SVG to the DOM so we have maximum ability to manipulate it

Reply:
also it probably wasn't baked in before because the `renderSVGDOM` prop was implemented a while after the `icon` prop for the `Button`. so it probably was just not backfilled